README.md

```diff
@@ -28,13 +27,29 @@ To help people get started with coding as a daily habit. Aim is to stay consiste
 
     a. Create different directories for each day, i.e. Day_1, Day_2, Day_3 ... and so on.
 
-    b. Push all the code into your main directory. (The one you created with your usedname) 
+    b. Push all the code into your main directory. (The one you created with your usedname)
 
     Do any of the 3(a) or 3(b)
 
+---
 
 
-
-
----
-_Maintainers_ : [Saurabh](https://github.com/srbhr), [Himanshu](https://github.com/himanshu007-creator) , [Mahi](https://github.com/mahimonga), [Soham](https://github.com/soham117) 
\ No newline at end of file
+### FAQ
```

Comment:
This whole part is still not good to view @soham117 !
Lemme share it here with you what you should do.

Reply:
![image](https://user-images.githubusercontent.com/54472864/149608418-4cdf00be-8d9a-4cd9-84b2-849697f3b9d2.png)
It's looking like this on Github. Try changing this via VScode, else I'll do it!